app/views/hunter_backstories/_hunter_backstory.html.erb

```diff
@@ -1,8 +1,8 @@
 <% if hunter_backstory %>
-  <h4><%= playbook.backstory&.dig(:name) %></h4>
+  <h4><%= link_to hunter_backstory.backstory&.name, hunter_backstory %></h4>
```

Comment:
If `backstory` is `nil` (which apparently is possible, since we are using `&`), then won't this line become
```
<%= link_to nil, hunter_backstory %>
```
So there won't be any text for the link, and nothing to click on?